packages/itwin/tree-widget/README.md

```diff
@@ -2,60 +2,177 @@
 
 Copyright © Bentley Systems, Incorporated. All rights reserved.
 
-The tree-widget-react package provides React components to access Models and Categories within an iModel via a UiProvider, `TreeWidgetUiItemsProvider`.
+The `@itwin/tree-widget-react` package provides React components to build a widget with tree components' selector, along with all the building blocks that can be used individually.
 
-The package also provides the underlying component, `TreeWidgetComponent`, which you can wrap within your own custom UiProvider and pass in your own custom trees to display.
+![Widget example](./media/widget.png)
```

Comment:
i think for this to show up correclty on npmjs, you'll need to reference the absolute path when the pr is merged into master

so https://github.com/iTwin/viewer-components-react/blob/master/packages/itwin/tree-widget/media/widget.png

Reply:
yes, we're tracking that with this issue: https://github.com/iTwin/viewer-components-react/issues/450